samples/wasm/README.md

```diff
@@ -0,0 +1,22 @@
+# Sample Wasm
+
+## DISCLAIMER
+
+> This is a sample application with demonstration purpose only,
+> do not use it with real addresses, real transactions, or real funds.
+> Use it at your own risk.
+
+## Prerequisites
+
+1. Everything you need to build the wallet core: https://developer.trustwallet.com/wallet-core/developing-the-library/building
+2. Install [emsdk](https://emscripten.org/docs/getting_started/downloads.html)
+- - python3
+- - cmake
+- - run `tools/install-wasm-dependencies`
+
```

Comment:
Maybe add a step to check for variable EMSDK, and if not set, set it.

Reply:
good point, updated readme